README.md

```diff
@@ -68,39 +68,24 @@ npx ignite-cli@latest new PizzaApp
 npx ignite-cli@latest new PizzaApp --yes
 ```
 
-Once you're up and running, check out our new [Getting Started Guide](/docs/Guide.md) guide or the rest of our [docs](https://github.com/infinitered/ignite/blob/master/docs).
+Once you're up and running, check out our [Getting Started Guide](https://docs.infinite.red/ignite-cli/Guide/).
 
-If you'd like to follow a tutorial, check out [this one from Robin Heinze](https://shift.infinite.red/creating-a-trivia-app-with-ignite-bowser-part-1-1987cc6e93a1). _Note that this was created for a previous version of Ignite -- we are working on updating it!_
-
-## Generators
-
-_The hidden gem of Ignite._ Generators help you scaffold your app very quickly, be it for a proof-of-concept, a demo, or a production app. Generators are there to save you time, keep your code consistent, and help you with the basic structure of your app.
-
-```
-npx ignite-cli generate --help
-```
-
-...will give you information on what generators are present. To learn more, check out our [Generators](/docs/concept//Generators.md) documentation.
+If you'd like to follow a tutorial, check out [this one from Robin Heinze](https://shift.infinite.red/creating-a-trivia-app-with-ignite-bowser-part-1-1987cc6e93a1).
 
 ### Troubleshooting
 
 The above commands may fail with various errors, depending on your operating system and dependency versions. Some troubleshooting steps to follow:
 
 - Uninstall global versions of the Ignite CLI via `npm uninstall -g ignite-cli` and use the CLI via `npx ignite-cli`
-- Make sure you are using a reasonably recent version of Node. This can be checked via the `node --version` command. If you require multiple Node versions on your system, install `nvm`, and then run `nvm install --lts`. At the time of writing, Node LTS is v16.x.x.
+- Make sure you are using a reasonably recent version of Node. This can be checked via the `node --version` command. If you require multiple Node versions on your system, install `nvm`, and then run `nvm install --lts`. At the time of writing, Node LTS is v20.x.x.
 - If the installation fails because of an Xcode error (missing Xcode command line tools), the easiest way to install them is to run `sudo xcode-select --install` in your terminal.
 - If Xcode and command line tools are already installed, but the installation complains about missing patch dependencies, you may need to switch the Xcode location to something else: `sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`
 - Opening the project in Xcode can give you other insights into what's happening: `open ./ios/<yourapp>.xcworkspace`
 - Add the `--debug` switch to the Ignite CLI new command to provide additional output during project initialization
 
-Note:
-
-- Navigation persistence is OFF by default in production.
-- Error boundary is set to 'always' display by default.
```

Comment:
I think the navigation persistence note is wrong, and the error boundary one seems unnecessary to me.